aws-predictions/src/main/java/com/amplifyframework/predictions/aws/adapter/IdentifyTextResultTransformers.java

```diff
@@ -0,0 +1,238 @@
+                box.getTop() + box.getHeight()
+        );
+    }
+
+
+    /**
+     * Converts geometric polygon from Amazon Textract into
+     * Amplify-compatible polygon object.
+     * @param polygon the polygon object by Amazon Textract
+     * @return the polygon object representing vertices
+     */
+    @Nullable
+    public static Polygon fromPoints(@Nullable List<Point> polygon) {
+        if (CollectionUtils.isNullOrEmpty(polygon)) {
+            return null;
+        }
+        List<PointF> points = new ArrayList<>();
+        for (Point point : polygon) {
+            PointF androidPoint = new PointF(
+                    point.getX(),
+                    point.getY()
+            );
+            points.add(androidPoint);
+        }
+        return Polygon.fromPoints(points);
+    }
+
+    /**
+     * Converts a given Amazon Textract block into Amplify-compatible
+     * table object. Returns null if not a valid table.
+     * @param block Textract text block
+     * @param blockMap map of Textract blocks by their IDs
+     * @return Amplify Table instance
+     */
+    @Nullable
+    public static Table processTable(@NonNull Block block, @NonNull Map<String, Block> blockMap) {
+        Objects.requireNonNull(block);
+        Objects.requireNonNull(blockMap);
+
+        if (!BlockType.TABLE.toString().equals(block.getBlockType())) {
+            return null;
+        }
+
+        RectF box = fromBoundingBox(block.getGeometry().getBoundingBox());
+        Polygon polygon = fromPoints(block.getGeometry().getPolygon());
+        List<Cell> cells = new ArrayList<>();
+        Set<Integer> rows = new HashSet<>();
+        Set<Integer> cols = new HashSet<>();
+        // Each TABLE block contains CELL blocks
+        for (Relationship relationship : block.getRelationships()) {
+            for (String cellId : relationship.getIds()) {
+                Block cellBlock = blockMap.get(cellId);
+                if (cellBlock == null) {
+                    continue;
+                }
+                rows.add(cellBlock.getRowIndex() - 1);
+                cols.add(cellBlock.getColumnIndex() - 1);
+                cells.add(processTableCell(cellBlock, blockMap));
+            }
+        }
+
+        return Table.builder()
+                .cells(cells)
+                .confidence(block.getConfidence())
+                .box(box)
+                .polygon(polygon)
+                .rowSize(rows.size())
+                .columnSize(cols.size())
+                .build();
+    }
+
+    /**
+     * Converts a given Amazon Textract block into Amplify-compatible
+     * key-value pair feature. Returns null if not a valid table.
+     * @param block Textract text block
+     * @param blockMap map of Textract blocks by their IDs
+     * @return Amplify Table instance
+     */
+    @Nullable
+    public static BoundedKeyValue processKeyValue(@NonNull Block block, @NonNull Map<String, Block> blockMap) {
```

Comment:
Hmm, these transformers are internal, so naming isn't all that important. But I agree that my naming has been somewhat inconsistent (mostly because I was doing `from<Something>`, but Textract is now requiring a bunch of conversion from `Block`, causing naming conflicts). 

Will start using `fetch<Something>` again, like I do in other services.